Tasks/PublishSymbols/task.json

```diff
@@ -48,18 +48,18 @@
             "helpMarkDown": "The path to the folder that is searched for symbol files.  The default is the configured source folder."
         },
         {
-            "name": "SourceFolder",
-            "type": "filePath",
-            "label": "Path to source folder",
-            "defaultValue": "",
+            "name": "TreatNotIndexedAsWarning",
```

Comment:
This is a new input requested by Chris. I already added the logic in a previous pull request. Now this finally adds the input to the UI.